server/authentication/passport.ts

```diff
@@ -11,6 +11,16 @@ import User from '../models/hmppsAuth/user'
 
 const authConfig = config.apis.hmppsAuth
 
+// the following code defines the Express.User type which allows @types/passport
+// to correctly augment Express.Request with the LoggedInUser created below
```

Comment:
Would it make sense to put this narrative in the commit message?

Reply:
done!